--- a/component/common/network/ssl/mbedtls-2.4.0/include/mbedtls/config.h	2017-11-17 13:22:05.929341100 +0800
+++ b/component/common/network/ssl/mbedtls-2.4.0/include/mbedtls/config.h	2017-11-17 13:50:56.833428200 +0800
@@ -2,7 +2,9 @@
 
 #include "rom_ssl_ram_map.h"
 #define RTL_HW_CRYPTO
-//#define SUPPORT_HW_SW_CRYPTO
+#define SUPPORT_HW_SW_CRYPTO
+#define MBEDTLS_PADLOCK_C
+#define MBEDTLS_HAVE_ASM
 #define RTL_CRYPTO_FRAGMENT               15360 /* 15*1024 < 16000 */
 
 #if CONFIG_SSL_RSA
