--- a/component/common/network/lwip/lwip_v1.4.1/src/core/dhcp.c	2017-11-17 13:22:05.694966100 +0800
+++ b/component/common/network/lwip/lwip_v1.4.1/src/core/dhcp.c	2017-11-17 13:50:54.833428200 +0800
@@ -949,6 +949,11 @@
   return result;
 }
 
+static void (*dhcp_bound_fn)(int interface, ip_addr_t *ipaddr);
+void dhcp_set_bound_handler(void (*handler)(int, ip_addr_t *))
+{
+	dhcp_bound_fn = handler;
+}
 
 /**
  * Bind the interface to the offered IP address.
@@ -1054,6 +1059,8 @@
   netif_set_gw(netif, &gw_addr);
   /* bring the interface up */
   netif_set_up(netif);
+  if (dhcp_bound_fn)
+  	dhcp_bound_fn(netif->num, &dhcp->offered_ip_addr);
   /* netif is now bound to DHCP leased address */
   dhcp_set_state(dhcp, DHCP_BOUND);
 }
