--- a/component/os/freertos/freertos_v8.1.2/Source/portable/MemMang/heap_5.c	2017-11-17 13:22:06.257466100 +0800
+++ b/component/os/freertos/freertos_v8.1.2/Source/portable/MemMang/heap_5.c	2017-11-17 13:50:59.708428200 +0800
@@ -175,7 +175,7 @@
 {
 	{ (uint8_t*)0x10002300, 0x3D00 },	// Image1 recycle heap
 	{ ucHeap, sizeof(ucHeap) }, 		// Defines a block from ucHeap
-#if 0
+#if 1
 	{ (uint8_t*)0x301b5000, 300*1024 }, // SDRAM heap
 #endif        
 	{ NULL, 0 } 							// Terminates the array.
