--- a/component/soc/realtek/8195a/misc/platform/ota_8195a.c	2017-11-17 13:22:06.335591100 +0800
+++ b/component/soc/realtek/8195a/misc/platform/ota_8195a.c	2017-11-17 13:51:00.829521900 +0800
@@ -105,7 +105,7 @@
 	uint32_t IMAGE_x = 0, ImgxLen = 0, ImgxAddr = 0;
 	uint32_t NewImg2Addr = 0; 
 #if WRITE_OTA_ADDR
-	uint32_t ota_addr = 0x80000;
+	uint32_t ota_addr = 0x100000;
 #endif
 	
 	DBG_INFO_MSG_OFF(_DBG_SPI_FLASH_);
