--- a/component/common/network/ssl/mbedtls-2.4.0/library/pkparse.c	2017-11-17 13:22:06.101216100 +0800
+++ b/component/common/network/ssl/mbedtls-2.4.0/library/pkparse.c	2017-11-17 13:50:58.208428200 +0800
@@ -506,7 +506,7 @@
  *      publicExponent    INTEGER   -- e
  *  }
  */
-static int pk_get_rsapubkey( unsigned char **p,
+int pk_get_rsapubkey( unsigned char **p,
                              const unsigned char *end,
                              mbedtls_rsa_context *rsa )
 {
