--- a/project/realtek_ameba1_va0_example/inc/platform_opts.h	2017-11-17 13:22:06.569966100 +0800
+++ b/project/realtek_ameba1_va0_example/inc/platform_opts.h	2017-11-17 13:51:02.532646900 +0800
@@ -50,9 +50,9 @@
 /**
 * For common flash usage  
 */
-#define AP_SETTING_SECTOR		0x000FE000
-#define UART_SETTING_SECTOR		0x000FC000
-#define FAST_RECONNECT_DATA 	(0x80000 - 0x1000)
+#define AP_SETTING_SECTOR		0x000F6000
+#define UART_SETTING_SECTOR		0x000F4000
+#define FAST_RECONNECT_DATA		0x000F3000
 
 /**
  * For Wlan configurations
@@ -98,8 +98,8 @@
 #endif
 
 /* For SSL/TLS */
-#define CONFIG_USE_POLARSSL     1
-#define CONFIG_USE_MBEDTLS      0
+#define CONFIG_USE_POLARSSL     0
+#define CONFIG_USE_MBEDTLS      1
 #if ((CONFIG_USE_POLARSSL == 0) && (CONFIG_USE_MBEDTLS == 0)) || ((CONFIG_USE_POLARSSL == 1) && (CONFIG_USE_MBEDTLS == 1))
 #undef CONFIG_USE_POLARSSL
 #define CONFIG_USE_POLARSSL 1
@@ -111,7 +111,7 @@
 #define CONFIG_INCLUDE_SIMPLE_CONFIG		1
 
 /*For fast reconnection*/
-#define CONFIG_EXAMPLE_WLAN_FAST_CONNECT	1
+#define CONFIG_EXAMPLE_WLAN_FAST_CONNECT	0
 
 /*For wowlan service settings*/
 #define CONFIG_WOWLAN_SERVICE           			0
