--- a/project/realtek_ameba1_va0_example/EWARM-RELEASE/Project.ewp	2017-11-17 13:22:06.429341100 +0800
+++ b/project/realtek_ameba1_va0_example/EWARM-RELEASE/Project.ewp	2017-11-17 13:51:04.361748500 +0800
@@ -48,11 +48,11 @@
         <option>
           <name>Output variant</name>
           <version>2</version>
-          <state>7</state>
+          <state>1</state>
         </option>
         <option>
           <name>Output description</name>
-          <state>No specifier a, A, no specifier n, no float nor long long, no flags.</state>
+          <state>Full formatting.</state>
         </option>
         <option>
           <name>GOutputBinary</name>
@@ -3016,6 +3016,451 @@
     </settings>
   </configuration>
   <group>
+    <name>ayla</name>
+    <group>
+      <name>Demo</name>
+      <excluded>
+        <configuration>INIC_GSPI_H</configuration>
+      </excluded>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\application\ayla_demo\src\demo_conf.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\application\ayla_demo\src\demo_common.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\application\ayla_demo\src\demo_ledevb.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\application\ayla_demo\src\demo_sched.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\application\ayla_demo\src\demo_wifi.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\application\ayla_demo\src\demo_stub.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\application\ayla_demo\src\ota.c</name>
+      </file>
+    </group>
+    <group>
+      <name>libada</name>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\ada_conf.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\ada_err.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\client.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\client_auth.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\client_conf.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\client_ota.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\client_prop.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\conf_json.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\lan.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\log_page.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\notify.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\oem.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\prop_mgr.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\prop_page.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\sched.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\al\ameba\sched_conf.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\schedeval.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\server_mem.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\server_req.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\sprop.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\sys_conf.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\al\ameba\ada_lock.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\al\ameba\ameba_conf.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\al\ameba\ameba_log.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\al\ameba\client_task.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\al\ameba\conf_table.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\al\ameba\http_client.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\al\ameba\notify_task.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\al\ameba\server.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\al\ameba\stubs.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\certs\ca_certs_der.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\text\custom.css.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\text\regtoken.html.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\dnss.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libada\dnss_util.c</name>
+      </file>
+    </group>
+    <group>
+      <name>libayla</name>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\arch\ameba\clock.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\arch\ameba\conf_flash.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\arch\ameba\malloc.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\clock_fmt.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\clock_sw.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\clock_utils.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\cmd_handle.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\conf.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\crc.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\crc16.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\crc8.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\endian.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\format_mac.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\format_string.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\hostname_valid.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\http.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\ext\jsmn\jsmn.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\jsmn_get.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\jsmn_iterate.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\json_format_string.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\log.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\log_cmd.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\log_conf.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\log_io.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\lookup_by_name.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\lookup_by_val.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\parse_argv.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\parse_date.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\parse_hex.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\parse_mac.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\parse_url.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\timer.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\uri_arg_allowed_map.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\uri_decode.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\uri_encode.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\utf8.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\xml.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\xml_decode.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\xml_encode.c</name>
+      </file>
+    </group>
+    <group>
+      <name>libnet</name>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libnet\ameba\ssl.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libnet\ameba\net_realtek.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libnet\ameba\net_mbuf.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libnet\ameba\ipaddr_fmt.c</name>
+      </file>
+    </group>
+    <group>
+      <name>libadw</name>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libadw\wifi.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libadw\adw_wifi_conf.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libadw\wifi_page.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libadw\wifi_ameba.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libadw\text\lock.gif.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libadw\text\refresh.gif.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libadw\text\wifi.html.c</name>
+      </file>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libadw\text\wifi.js.c</name>
+      </file>
+    </group>
+    <configuration>
+      <name>Debug</name>
+      <toolchain>
+        <name>ARM</name>
+      </toolchain>
+      <debug>1</debug>
+      <settings>
+        <name>ICCARM</name>
+        <archiveVersion>2</archiveVersion>
+        <data>
+          <version>31</version>
+          <wantNonLocal>1</wantNonLocal>
+          <debug>1</debug>
+          <option>
+            <name>IExtraOptions</name>
+            <state>--section .rodata=.sdram.data</state>
+          </option>
+          <option>
+            <name>CCDefines</name>
+            <state>CONFIG_PLATFORM_8195A</state>
+            <state>AMEBA</state>
+            <state>AMEBA_IAR</state>
+            <state>STATIC_WEB_CONTENT_IN_MEMORY</state>
+            <state>LOG_SEV_SHORT</state>
+            <state>CLIENT_MT</state>
+            <state>_HAS_ASSERT_F_</state>
+            <state>AYLA_FreeRTOS</state>
+            <state>NO_LOG_BUF</state>
+            <state>LOG_LOCK</state>
+            <state>CONF_NO_ID_FILE</state>
+            <state>XXD_BIN_TO_C</state>
+            <state>AYLA_WIFI_SUPPORT</state>
+            <state>FLASH_CONF_CACHE</state>
+            <state>STATIC_WEB_CONTENT_IN_MEMORY</state>
+          </option>
+          <option>
+            <name>CCDiagSuppress</name>
+            <state>Be006,Pa050,Pa039,Pa089,Pe014,Pe068,Pe089,Pe167,Pe177,Pe186,Pe188,Pe375,Pe550,Pe550,Pe223,Pe549,Pe550,Pe301</state>
+          </option>
+          <option>
+            <name>CCIncludePath2</name>
+            <state>$PROJ_DIR$\..\..\..\component\common\ayla\include</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\ayla\libada</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\ayla\libada\include</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\ayla\libayla\include</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\ayla\ext\jsmn</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\ayla\libnet\include</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\ayla\libnet\ameba\include</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\ayla\libnet\lwip\include</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\ayla\libadw\include</state>
+            <state>$PROJ_DIR$\..\inc</state>
+            <state>$PROJ_DIR$\..\..\..\component\soc\realtek\common\bsp</state>
+            <state>$PROJ_DIR$\..\..\..\component\os\freertos</state>
+            <state>$PROJ_DIR$\..\..\..\component\os\freertos\freertos_v8.1.2\Source\include</state>
+            <state>$PROJ_DIR$\..\..\..\component\os\freertos\freertos_v8.1.2\Source\portable\IAR\ARM_CM3</state>
+            <state>$PROJ_DIR$\..\..\..\component\os\os_dep\include</state>
+            <state>$PROJ_DIR$\..\..\..\component\soc\realtek\8195a\misc\driver</state>
+            <state>$PROJ_DIR$\..\..\..\component\soc\realtek\8195a\misc\os</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\api\network\include</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\api</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\api\platform</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\api\wifi</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\api\wifi\rtw_wpa_supplicant\src</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\application</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\example</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\example\wlan_fast_connect</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\mbed\api</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\mbed\hal</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\mbed\hal_ext</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\mbed\targets\hal\rtl8195a</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\network</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\network\lwip\lwip_v1.4.1\port\realtek\freertos</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\network\lwip\lwip_v1.4.1\src\include</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\network\lwip\lwip_v1.4.1\src\include\lwip</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\network\lwip\lwip_v1.4.1\src\include\ipv4</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\network\lwip\lwip_v1.4.1\port\realtek</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\network\lwip\lwip_v1.4.1\src\include\posix</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\test</state>
+            <state>$PROJ_DIR$\..\..\..\component\soc\realtek\8195a\cmsis</state>
+            <state>$PROJ_DIR$\..\..\..\component\soc\realtek\8195a\cmsis\device</state>
+            <state>$PROJ_DIR$\..\..\..\component\soc\realtek\8195a\fwlib</state>
+            <state>$PROJ_DIR$\..\..\..\component\soc\realtek\8195a\fwlib\rtl8195a</state>
+            <state>$PROJ_DIR$\..\..\..\component\soc\realtek\8195a\misc\rtl_std_lib\include</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\drivers\wlan\realtek\include</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\drivers\wlan\realtek\src\osdep</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\drivers\wlan\realtek\src\hci</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\drivers\wlan\realtek\src\hal</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\drivers\wlan\realtek\src\hal\OUTSRC</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\drivers\video\realtek\common</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\drivers\i2s</state>
+            <state>$PROJ_DIR$\..\..\..\component\soc\realtek\8195a\fwlib\ram_lib\wlan\realtek\wlan_ram_map\rom</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\network\ssl\polarssl-1.3.8\include</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\network\ssl\mbedtls-2.4.0\include</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\network\ssl\ssl_ram_map\rom</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\utilities</state>
+            <state>$PROJ_DIR$\..\..\..\component\soc\realtek\8195a\misc\rtl_std_lib\include</state>
+            <state>$PROJ_DIR$\..\..\..\component\soc\realtek\8195a\fwlib\ram_lib\usb_otg\include</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\video\v4l2\inc</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\audio</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\media\rtp_codec</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\drivers\usb_class\host\uvc\inc</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\drivers\usb_class\device</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\drivers\usb_class\device\class</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\file_system\fatfs</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\file_system\fatfs\r0.10c\include</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\drivers\sdio\realtek\sdio_host\inc</state>
+            <state>$PROJ_DIR$\..\..\..\component\soc\realtek\8195a\misc\platform</state>
+            <state>$PROJ_DIR$\..\..\..\component\soc\realtek\8195a\misc\fpb</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\media\framework</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\media\codec\mp3</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\media\rtp_codec\mjpeg</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\media\muxer</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\application\mqtt\MQTTClient</state>
+            <state>$PROJ_DIR$\..\..\..\component\common\ayla\libadw\ameba</state>
+          </option>
+          <option>
+            <name>CCCodeSection</name>
+            <state>.sdram.text</state>
+          </option>
+          <option>
+            <name>IExtraOptionsCheck</name>
+            <state>1</state>
+          </option>
+          <option>
+            <name>IExtraOptions</name>
+            <state>--section .rodata=.sdram.data</state>
+          </option>
+        </data>
+      </settings>
+    </configuration>
+  </group>
+  <group>
+    <name>ayla-ameba</name>
+    <group>
+      <name>Demo</name>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\application\ayla_demo\src\led_key.c</name>
+      </file>
+    </group>
+    <group>
+      <name>libadw</name>
+      <file>
+        <name>$PROJ_DIR$\..\..\..\component\common\ayla\libadw\wifi_ameba_drv.c</name>
+      </file>
+    </group>
+  </group>
+  <group>
     <name>cmsis</name>
     <file>
       <name>$PROJ_DIR$\..\..\..\component\soc\realtek\8195a\cmsis\device\system_8195a.c</name>
@@ -3289,45 +3734,6 @@
       <excluded>
         <configuration>INIC-SDIO</configuration>
       </excluded>
-      <group>
-        <name>mqtt</name>
-        <file>
-          <name>$PROJ_DIR$\..\..\..\component\common\application\mqtt\MQTTClient\MQTTClient.c</name>
-        </file>
-        <file>
-          <name>$PROJ_DIR$\..\..\..\component\common\application\mqtt\MQTTPacket\MQTTConnectClient.c</name>
-        </file>
-        <file>
-          <name>$PROJ_DIR$\..\..\..\component\common\application\mqtt\MQTTPacket\MQTTConnectServer.c</name>
-        </file>
-        <file>
-          <name>$PROJ_DIR$\..\..\..\component\common\application\mqtt\MQTTPacket\MQTTDeserializePublish.c</name>
-        </file>
-        <file>
-          <name>$PROJ_DIR$\..\..\..\component\common\application\mqtt\MQTTPacket\MQTTFormat.c</name>
-        </file>
-        <file>
-          <name>$PROJ_DIR$\..\..\..\component\common\application\mqtt\MQTTClient\MQTTFreertos.c</name>
-        </file>
-        <file>
-          <name>$PROJ_DIR$\..\..\..\component\common\application\mqtt\MQTTPacket\MQTTPacket.c</name>
-        </file>
-        <file>
-          <name>$PROJ_DIR$\..\..\..\component\common\application\mqtt\MQTTPacket\MQTTSerializePublish.c</name>
-        </file>
-        <file>
-          <name>$PROJ_DIR$\..\..\..\component\common\application\mqtt\MQTTPacket\MQTTSubscribeClient.c</name>
-        </file>
-        <file>
-          <name>$PROJ_DIR$\..\..\..\component\common\application\mqtt\MQTTPacket\MQTTSubscribeServer.c</name>
-        </file>
-        <file>
-          <name>$PROJ_DIR$\..\..\..\component\common\application\mqtt\MQTTPacket\MQTTUnsubscribeClient.c</name>
-        </file>
-        <file>
-          <name>$PROJ_DIR$\..\..\..\component\common\application\mqtt\MQTTPacket\MQTTUnsubscribeServer.c</name>
-        </file>
-      </group>
       <file>
         <name>$PROJ_DIR$\..\..\..\component\soc\realtek\8195a\misc\platform\ota_8195a.c</name>
       </file>
@@ -3574,7 +3980,6 @@
       <group>
         <name>mbedtls</name>
         <excluded>
-          <configuration>Debug</configuration>
           <configuration>UVC</configuration>
         </excluded>
         <file>
@@ -3583,6 +3988,9 @@
       </group>
       <group>
         <name>polarssl</name>
+        <excluded>
+          <configuration>Debug</configuration>
+        </excluded>
         <file>
           <name>$PROJ_DIR$\..\..\..\component\common\network\ssl\polarssl-1.3.8\library\bignum.c</name>
         </file>
@@ -3846,7 +4254,6 @@
       <group>
         <name>mbedtls</name>
         <excluded>
-          <configuration>Debug</configuration>
           <configuration>UVC</configuration>
         </excluded>
         <file>
@@ -4062,6 +4469,9 @@
       </group>
       <group>
         <name>polarssl</name>
+        <excluded>
+          <configuration>Debug</configuration>
+        </excluded>
         <file>
           <name>$PROJ_DIR$\..\..\..\component\common\network\ssl\polarssl-1.3.8\library\aes.c</name>
         </file>
@@ -5184,122 +5594,11 @@
     <group>
       <name>example</name>
       <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\amazon_awsiot\example_amazon_awsiot.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\audio_mp3\example_audio_mp3.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\bcast\example_bcast.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\eap\example_eap.c</name>
-      </file>
-      <file>
         <name>$PROJ_DIR$\..\..\..\component\common\example\example_entry.c</name>
       </file>
       <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\fatfs\example_fatfs.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\get_beacon_frame\example_get_beacon_frame.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\googlenest\example_google.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\high_load_memory_use\example_high_load_memory_use.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\http_client\example_http_client.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\http_download\example_http_download.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\httpc\example_httpc.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\httpd\example_httpd.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\mcast\example_mcast.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\mdns\example_mdns.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\media_audio_from_rtp\example_media_audio_from_rtp.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\media_single_stream\example_media_ss.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\media_time_lapse\example_media_tl.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\nonblock_connect\example_nonblock_connect.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\ota_http\example_ota_http.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\pppoe\example_pppoe.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\rarp\example_rarp.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\sntp_showtime\example_sntp_showtime.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\socket_select\example_socket_select.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\socket_tcp_trx\example_socket_tcp_trx_1.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\socket_tcp_trx\example_socket_tcp_trx_2.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\spi_atcmd\example_spi_atcmd.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\ssl_download\example_ssl_download.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\ssl_server\example_ssl_server.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\tcp_keepalive\example_tcp_keepalive.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\uart_atcmd\example_uart_atcmd.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\uart_firmware_update\example_uart_update.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\usb_mass_storage\example_usb_msc.c</name>
-        <excluded>
-          <configuration>UVC</configuration>
-        </excluded>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\wifi_mac_monitor\example_wifi_mac_monitor.c</name>
-      </file>
-      <file>
         <name>$PROJ_DIR$\..\..\..\component\common\example\wlan_fast_connect\example_wlan_fast_connect.c</name>
       </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\wlan_scenario\example_wlan_scenario.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\websocket\example_wsclient.c</name>
-      </file>
-      <file>
-        <name>$PROJ_DIR$\..\..\..\component\common\example\xml\example_xml.c</name>
-      </file>
     </group>
     <group>
       <name>FatFs</name>
@@ -5335,12 +5634,6 @@
       </group>
     </group>
     <file>
-      <name>$PROJ_DIR$\..\..\..\component\common\utilities\cJSON.c</name>
-    </file>
-    <file>
-      <name>$PROJ_DIR$\..\..\..\component\common\utilities\http_client.c</name>
-    </file>
-    <file>
       <name>$PROJ_DIR$\..\..\..\component\common\application\xmodem\uart_fw_update.c</name>
     </file>
     <file>
@@ -5349,9 +5642,6 @@
     <file>
       <name>$PROJ_DIR$\..\..\..\component\common\utilities\webserver.c</name>
     </file>
-    <file>
-      <name>$PROJ_DIR$\..\..\..\component\common\utilities\xml.c</name>
-    </file>
   </group>
 </project>
 
