--- a/component/common/network/lwip/lwip_v1.4.1/src/api/sockets.c	2017-11-17 13:22:05.616841100 +0800
+++ b/component/common/network/lwip/lwip_v1.4.1/src/api/sockets.c	2017-11-17 13:50:54.177178200 +0800
@@ -2443,4 +2443,15 @@
 *                           Added  by Realtek        end                    *
 **************************************************************/
 
+struct tcp_pcb *sock_get_pcb(int s)
+{
+  struct lwip_sock *sock;
+
+  sock = tryget_socket(s);
+  if (!sock) {
+    return NULL;
+  }
+  return (struct tcp_pcb *)sock->conn->pcb.tcp;
+}
+
 #endif /* LWIP_SOCKET */
