--- a/component/common/api/network/src/wlan_network.c	2017-11-17 13:22:05.476216100 +0800
+++ b/component/common/api/network/src/wlan_network.c	2017-11-17 13:50:52.724053200 +0800
@@ -32,6 +32,9 @@
 
 xSemaphoreHandle uart_rx_interrupt_sema = NULL;
 
+void ayla_demo_init(void);
+void ayla_wlan_init(void);
+
 void init_thread(void *param)
 {
 
@@ -44,6 +47,7 @@
 #if CONFIG_WIFI_IND_USE_THREAD
 	wifi_manager_init();
 #endif
+        ayla_wlan_init();
 #if CONFIG_WLAN
 	wifi_on(RTW_MODE_STA);
 #if CONFIG_AUTO_RECONNECT
@@ -60,6 +64,9 @@
 	start_interactive_mode();
 #endif	
 
+	printf("\n");
+	ayla_demo_init();
+
 	/* Kill init thread after all init tasks done */
 	vTaskDelete(NULL);
 }
